.circleci/config.yml

```diff
@@ -28,14 +28,44 @@ jobs:
           name: run tests
           command: |
             bash scripts/run_backend_tests.sh --generate_coverage_report
+    
+      - save_cache:
```

Comment:
What's going on here (and why are both paths the same)?

Reply:
So basically CircleCI encourage versioning caches. 
https://circleci.com/docs/2.0/caching/#source-caching
Due to the lack of a package.json (and therefore I cannot pipe`checksum package.json` here), I use this primitive v1.01 naming scheme. 
Whenever I add a new `install_node_module` commands, I would just change the version number and invalidate CircleCI cache. 
I figured there should be a cleaner way of doing this. Should I be `checksum`-ing the bash script file(run_linter.sh, setup.sh,etc,...)?
Yea the folder path is duplicated, will patch that asap.

Reply:
Chatted offline -- best course of action is probably to bring in package.json and get rid of install_node_module.

Reply:
Yep, will replace `install_node_module` with package.json in a separate PR. Revert these changes for now 👍